src/src.pro

```diff
@@ -715,7 +716,8 @@ else:ios {
             platforms/ios/ioscontroller.h \
             platforms/ios/iosdatamigration.h \
             platforms/ios/iosnotificationhandler.h \
-            platforms/ios/iosutils.h
+            platforms/ios/iosutils.h \
+            platforms/ios/iosadjusthelper.h
```

Comment:
alphabetic order

Reply:
I put it at the top right below `iosiaphandler.h`